kythe/cxx/indexer/cxx/testdata/df/df_binary.cc

```diff
@@ -0,0 +1,18 @@
+// Accumulator-style operators are in the influence graph.
+
+void f(int x, int y, int z) {
+    //- @x ref VarX
+    //- @x ref/writes VarX
+    //- @y ref VarY
+    //- !{ @y ref/writes VarY }
+    x += y;
+    //- @z ref VarZ
+    //- @z ref/writes VarZ
+    //- @x ref VarX
+    //- !{ @x ref/writes VarX }
+    z -= x;
+    //- VarX influences VarX
+    //- VarZ influences VarZ
+    //- VarY influences VarX
+    //- VarX influences VarZ
+}
```

Comment:
Is it worth a test for

  z -= x++

and the like?

Reply:
PTAL; I'm not sure if you have a different case in mind.

Reply:
LGTM